lib/src/request/CommonMediaRequest.ts

```diff
@@ -12,46 +13,51 @@ export type CommonMediaRequest = {
 	/**
 	 * The URL of the request.
 	 */
-	url: string
+	url: string;
 
 	/**
 	 * The request's method (GET, POST, etc).
 	 */
-	method: string
+	method?: string;
+
+	/**
+	 * The body of the request.
+	 */
+	body?: BodyInit;
 
 	/**
 	 * The response type with which the response from the server shall be compatible.
 	 */
-	responseType?: string
+	responseType?: RequestType;
```

Comment:
Having a more strict type for `responseType` allows us to map the `data` property of `CommonMediaResponse` to a specific data type.